code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php

```diff
@@ -40,7 +40,7 @@ private function browseLocation(Location $location, OutputInterface $output, int
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $locationId = $input->getArgument('locationId');
+        $locationId = (int) $input->getArgument('locationId');
```

Comment:
@mnocon Why PHPStan didn't see an issue here?

https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/Input/InputInterface.php#L83-L90 declares returning `mixed` but in fact it's probably only `int|array`.

Reply:
This one would be detected using level 9:
https://phpstan.org/user-guide/rule-levels

Bumping it results in:
```
  Line   api/public_php_api/src/Command/BrowseLocationsCommand.php                                                                     
 ------ ------------------------------------------------------------------------------------------------------------------------------ 
  :45    Parameter #1 $locationId of method Ibexa\Contracts\Core\Repository\LocationService::loadLocation() expects int, mixed given.  
 ------ ------------------------------------------------------------------------------------------------------------------------------ 
```

(and another 147 errors)

Reply:
We can think if we want to introduce this (level 10 is the highest right now), in product code we use level 8

Reply:
@mnocon I don't fix 147 errors here 😛. I would say that we don't need it in CI. I'm curious about those level 9 new errors. But there is already a hundred to check in the actual level 8 phpstan-baseline.neon 😅